.golangci.yml

```diff
@@ -32,13 +32,15 @@ linters:
     - goprintffuncname
     - gosec
     - gosimple
+    - gosmopolitan
     - govet
     - grouper
     - importas
     - ineffassign
     - interfacebloat
     - logrlint
     - makezero
+    - mirror
```

Comment:
Checks for unnecessary string/[]byte conversions when methods exist that don't require a conversion.